guides/how-to-readme.md

```diff
@@ -0,0 +1,77 @@
+
+Be sure to answer these questions in your description:
+
+* What is this project?
+* How does it work?
+* Who are the users?
+* What is the benefit for the users?
+* What is the goal of this project?
+
+Start with the most important things so people can continue their search if this project is not what they are looking for.
+
+Make this section shine: If your project has a front-end include a screenshot here; if there is a demo or there are live examples, include links; if there are features, list them.
+
+## 2. How to install, configure, run and deploy
+
+The basics a user needs to know to use the project. Be sure to document every necessary step and some common edge cases to get started and point to relevant further documentation if available. 
+
+These sections can be quite technical. You don’t need to explain everything as and you can expect users to search the web for what they don’t understand.
+
+> We find it works really well if you follow a two-step approach to develop the content for this section: first, help someone setup the site who has never done it before, and then write down the exact instructions. Next, ask someone to follow those instructions and see if you’ve missed anything. — [18F: Making `README`s Readable](https://open-source-guide.18f.gov/making-readmes-readable/)
+
+Make sure that these are always up-to-date. If you change functionality in the code, update the `README` concurrently.
+
+## 3. How to develop and test
+
+Everything a developer needs to know to work on this project, including style guides, how tests work and how to use the development tools.
+
+## 4. How to contribute and ask questions
+
+Here you can describe how to get involved in the project, contribute and ask questions. 
+
+If you have a `CONTRIBUTING` file, be sure to point to it from your `README`.
+
+## Other things to include
+
+Add more information if you think it is necessary as long as the `README` stays a document that is readable by every new user. For instance, if your install guide becomes very long consider creating a separate guide for this in the form of an `INSTALL` file or a guide in your documentation.
+
+Depending on your project you might also need to explain more about:
+
+* Nomenclature: what words are used in the code and what do they mean
+* Architecture: what is the structure you’ve decided on, and what do developers need to know about this
+* Dependencies: what software does this project depend on
+* Licence: what licence the project has and point to the `LICENCE` file.
+
+---
+
+## Inspirations and further reading
+
+* [18Fs Open Source Guide on `README`s](https://open-source-guide.18f.gov/making-readmes-readable/)
+* [Jesse Luotos `README` best practices](https://github.com/jehna/readme-best-practices/)
+* [`README`s for GOV.UK applications](https://docs.publishing.service.gov.uk/manual/readmes.html)
+* [How to write a good `README` on StackOverflow](https://stackoverflow.com/questions/2304863/how-to-write-a-good-readme)
```

Comment:
I would propose to start every paragraph of this document with a catching quote (like the one from Pursell). It catches attention and improves the focus of the reader. It's like an appetizer

Reply:
I like this stylistically, however, I'll add it as an issue for possible enhancement, since it requires some new effort